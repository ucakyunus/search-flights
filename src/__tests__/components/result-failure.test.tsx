import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import ResultFailure from '@/components/result/failure';

test('ResultFailure matches snapshot', () => {
  const { asFragment } = render(
    <Router>
      <ResultFailure />
    </Router>,
  );
  expect(asFragment()).toMatchSnapshot();
});
