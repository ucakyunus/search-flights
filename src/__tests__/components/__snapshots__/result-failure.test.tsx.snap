// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultFailure matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-j7qwjs"
  >
    <div
      class="MuiBox-root css-1ri905o"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-colorError MuiSvgIcon-fontSizeMedium css-1a4ayes-MuiSvgIcon-root"
        data-testid="CancelIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2m5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12z"
        />
      </svg>
      <p
        class="MuiTypography-root MuiTypography-body1 css-zp3nss-MuiTypography-root"
      >
        Kabin seciminiz tamamlanamadi.
      </p>
    </div>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedError MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorError MuiButton-root MuiButton-contained MuiButton-containedError MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorError css-1pqrzrr-MuiButtonBase-root-MuiButton-root"
      tabindex="0"
      type="button"
    >
      Başa Dön
    </button>
  </div>
</DocumentFragment>
`;
